chromadb/test/property/test_embeddings.py

```diff
@@ -114,18 +114,18 @@ def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID
             record_set
         )
 
-        if len(normalized_record_set["ids"]) > 0:
+        ids = normalized_record_set["ids"] or []
```

Comment:
This logic would be clearer if we were clear about the possible branches:
- There are already IDs in the `record_set_state` and the new `record_set` IDs are not `None` - possible intersection of IDs
- There are no IDs in the `record_set_state` and the `record_set` IDs are not `None` - purely additive, with IDs present
- There are already IDs in the `record_set_state` and the `record_set` IDs are `None` - purely additive, IDs are generate during `add`
- There are no IDs in the `record_set_state` and the `record_set` IDs are `None` - purely additive, IDs are generated during `add`

We should restructure this to make it clear.

Reply:
resolved offline

Reply:
i will leave it as it is since found intersected ids vs not found intersected ids is easier to understand than the 4 scenarios imo